lib/CL/pocl_ipc_mutex.h

```diff
@@ -50,7 +50,7 @@ int pocl_ipc_mutex_create (const char *name, pocl_ipc_mutex_t *ipc_mtx);
  * has locked the mutex. The mutex is unlocked in case the process
  * holding the lock terminates without unlocking first.
  *
- * The '�pc_mtx' must be a valid object created by pocl_ipc_mutex_create().
+ * The 'ípc_mtx' must be a valid object created by pocl_ipc_mutex_create().
```

Comment:
Hm encoding broken due to some incapable editor?

Reply:
Fixed.